app.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 require 'dotenv/load'
-require 'active_support/core_ext/hash/indifferent_access'
-require 'active_support/core_ext/object/to_query'
```

Comment:
Were these changes related to Rubocop? I see that it still works without it, but I feel like this leaves us with an implicit assumption that one of our other dependencies will be making these methods available (`json-jwt`), which could easily break in the future. I think having it explicitly required like it had been was a good idea.

Reply:
I see your point; I'll investigate requiring a gem (json/jwt) and its dependencies. Also, I see there could be a missing test; as you point out, would then break app.  Thanks for approving.  Let me gain more confidence in removing those 2 require lines.